### PR TITLE
Feat/homepage quickstart merge

### PR DESCRIPTION
**Pull request by johnnymatthews**
Adds Quickstart directly onto the homepage.

### CODE REVIEW

**Comment by filecorgi** [comment]
- Image optimization came back clean!
- Vuepress build was successful!

**Comment by filecorgi** [comment]
- Image optimization came back clean!
- Vuepress build was successful!

**Comment by filecorgi** [comment]
- Image optimization came back clean!
- Vuepress build was successful!

File: docs/README.md [hunk 1]
```diff
@@ -1,43 +1,144 @@
...
 title: Welcome to Web3.Storage
-description: Lorem ipsum.
+description: "Web3.Storage lets you store stuff on the decentralize internet without having to mess around or learn complex stuff."
 ---
 
- # Welcome to Web3.Storage
+# Welcome to Web3.Storage
 
-Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.
+Web3.Storage lets you store stuff on the decentralize internet without having to mess around or learn complex stuff.
```
Comment (yusefnapora) [inline docs/README.md:8]: lol, I do hate messing around with complex stuff 😄

File: docs/README.md [hunk 2]
```diff
@@ -1,43 +1,144 @@
...
+            const token = args.token
+
+            if (!token) {
+                console.error('A token is needed. You can create one on https://web3.storage')
+                    return
+            }
+
+        const storage = new Web3Storage({ token })
+
+            const files = await getFilesFromPath('./fixtures')
```
Comment (yusefnapora) [inline docs/README.md:83]: That `./fixtures` path is in the examples repo, but it won't exist on their machine.

If you want, I can update the script to read in the path from the command line.

**Review by yusefnapora — APPROVED** [review]
Looks good, with the caveat about the fixture path thing in the example script, which we can fix separately.

File: docs/README.md (hunk 2 above)
Comment (johnnymatthews) [reply docs/README.md:83]: Jah, was planning to update this PR to have the user add in the `fixtures` folder, but it _sounds_ like you've already added something into the script to just have the user enter a path instead!

**Comment by filecorgi** [comment]
- Image optimization came back clean!
- Vuepress build was successful!

**Comment by filecorgi** [comment]
- Image optimization came back clean!
- Vuepress build was successful!

**Comment by filecorgi** [comment]
- Image optimization came back clean!
- Vuepress build was successful!

**Comment by filecorgi** [comment]
- Image optimization came back clean!
- Vuepress build was successful!

**Comment by filecorgi** [comment]
- Image optimization came back clean!
- Vuepress build was successful!

**Comment by filecorgi** [comment]
- Image optimization came back clean!
- Vuepress build was successful!

**Comment by filecorgi** [comment]
- Image optimization came back clean!
- Vuepress build was successful!

**Pull request merged**